components/FrontPage/Combined/finalSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import TBox from "../textbox";
+import Btn from "../button";
+
+export default function FinalSection() {
+  return (
+    <div className="flex h-[400px] w-screen justify-center bg-slate-200 dark:bg-slate-800">
```

Comment:
```suggestion
    <div className="flex w-screen justify-center bg-slate-200 dark:bg-slate-800">
```

Instead of absolute height, prefer using padding instead.